services/movie_session.py

```diff
@@ -0,0 +1,55 @@
+from db.models import MovieSession, Movie, CinemaHall
+
+from django.db.models import QuerySet
+
+from datetime import datetime
+
+
+def create_movie_session(
+        movie_show_time: datetime,
+        movie_id: int,
+        cinema_hall_id: int
+) -> MovieSession:
+    return MovieSession.objects.create(
+        show_time=movie_show_time,
+        movie=Movie.objects.get(id=movie_id),
+        cinema_hall=CinemaHall.objects.get(id=cinema_hall_id)
+    )
+
+
+def get_movies_sessions(
+        session_date: datetime = None
+) -> QuerySet[MovieSession]:
+    if session_date:
+        return MovieSession.objects.filter(show_time__date=session_date)
+    return MovieSession.objects.all()
+
+
+def get_movie_session_by_id(movie_session_id: int) -> MovieSession:
+    return MovieSession.objects.get(id=movie_session_id)
+
+
+def update_movie_session(
+        session_id: int,
+        show_time: datetime = None,
+        movie_id: int = None,
+        cinema_hall_id: int = None
+) -> MovieSession:
+    updated_movie_session = MovieSession.objects.filter(
+        id=session_id
+    )
+
+    if show_time:
+        updated_movie_session.update(show_time=show_time)
+    if movie_id:
+        movie = Movie.objects.get(id=movie_id)
+        updated_movie_session.update(movie=movie)
+    if cinema_hall_id:
+        cinema_hall = CinemaHall.objects.get(id=cinema_hall_id)
+        updated_movie_session.update(cinema_hall=cinema_hall)
+
+    return updated_movie_session
```

Comment:
The `update_movie_session` function should return a single `MovieSession` instance instead of a queryset. Consider using `get` to retrieve the updated instance.